WordPress/WordPressStatsWidgets/LockScreenWidgets/ViewProvider/LockScreenSingleStatWidgetViewProvider.swift

```diff
@@ -8,7 +8,8 @@ struct LockScreenSingleStatWidgetViewProvider: LockScreenStatsWidgetsViewProvide
         let mapper = LockScreenWidgetViewModelMapper(data: data)
         let viewModel = mapper.getLockScreenSingleStatViewModel(
             title: LocalizableStrings.viewsTitle,
-            dateRange: LocalizableStrings.todayWidgetTitle
+            dateRange: LocalizableStrings.todayWidgetTitle,
```

Comment:
If we choose this direction, then we don't need `dateRange`.

Reply:
Oh yeah, I forgot we don't need to toggle two layouts for now. 
Let me update it for simplifying the code, thanks